### PR TITLE
[JENKINS-65424] - Maven descriptor step arguments documentation

### PR DESCRIPTION
**Pull request by Onyimatics**
- [x] All [tests](https://ci.appveyor.com/project/jfrog-ecosystem/jenkins-artifactory-plugin) passed. If this feature is not already covered by the tests, I added new tests.
- [x] This pull request is created in the [jfrog/jenkins-artifactory-plugin](https://github.com/jfrog/jenkins-artifactory-plugin) repository.

- [x] Please describe what you did
- Created `help-pomFile.html` file to add `pomFile` argument's description.
- Created `help-version.html` file to add `version` argument's description. 
- Created `help-versionPerModule.html` file to add `versionPerModule` argument's description. 
- Created `help-failOnSnapshot.html` file to add `failOnSnapshot` argument's description. 
- Created `help-dryRun.html` file to add `dryRun` argument's description. 

- [x] Link to relevant issues in GitHub or Jira
- [JENKINS-65424](https://issues.jenkins.io/browse/JENKINS-65424)

  @MarkEWaite @oleg-nenashev @kwhetstone @StackScribe
-----

### CODE REVIEW

File: src/main/resources/org/jfrog/hudson/pipeline/scripted/steps/MavenDescriptorStep/help-pomFile.html [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<div>
+    If your workspace has the top-level <code>pom.xml</code> in somewhere other than the 1st module's root directory, specify the path (relative to the module root) here, such as <code>parent/pom.xml</code>.
+    If left empty, defaults to <code>pom.xml</code>.
```
Comment (kwhetstone) [inline src/main/resources/org/jfrog/hudson/pipeline/scripted/steps/MavenDescriptorStep/help-pomFile.html:3]: Definitely like that you included the default

File: src/main/resources/org/jfrog/hudson/pipeline/scripted/steps/MavenDescriptorStep/help-version.html [hunk 2]
```diff
@@ -0,0 +1,6 @@
+<div>
+    Specify the Maven version.
```
Comment (kwhetstone) [inline src/main/resources/org/jfrog/hudson/pipeline/scripted/steps/MavenDescriptorStep/help-version.html:2]: Is there a default here?

**Review by kwhetstone — COMMENTED** [review]
I'm ok with using the "set to true" which could also indicate this is a checkbox,

**Comment by eyalbe4** [comment]
@Onyimatics - 
Where will these docs comments be visible from?
Will you be able to share a screenshot or a reference?

**Comment by Onyimatics** [comment]
> @Onyimatics -
> Where will these docs comments be visible from?
> Will you be able to share a screenshot or a reference?

On [Artifactory Plugin](https://www.jenkins.io/doc/pipeline/steps/artifactory/), it would be very handy to get a description for each argument.
image

This is it on the local reference page.
image

**Comment by eyalbe4** [comment]
This PR is very old and must probably out of date. I'm therefore closing it. We can reopen the discussion around it if needed.

**Pull request closed without merging**